example/MyFirstDrawing/MyFirstDrawingUITests/BrushToolView/GroupEraseSizeViewUITests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  GroupEraseSizeViewUITests.swift
+//  MyFirstDrawingUITests
+//
+//  Created by Stant Macmini n04 on 09/07/19.
+//  Copyright © 2019 Stant 02. All rights reserved.
+//
+
+import XCTest
+import StantDrawing
+@testable import MyFirstDrawing
+
+class GroupEraseSizeViewUITests: XCTestCase {
+    let app = XCUIApplication()
+    
+    override func setUp() {
+        continueAfterFailure = false
+        
+        app.launch()
+        
+        let btnErase = app.buttons["eraserfullIdentifier"]
+        btnErase.tap()
+    }
+    
+    override func tearDown() {}
+    
+    func testClickOnGroupEraseSizeViewAndShowAllButtons() {
+        let eraseSizeReference = app.images.matching(identifier: "eraserfullReferenceIdentifier")
+        
+        let btnEraseSize1 = app.buttons["eraserSize1Identifier"]
+        let btnEraseSize2 = app.buttons["eraserSize2Identifier"]
```

Comment:
I was taking a look on this buttons and I was wondering if it was better to separate them to a different class. 
I'm not sure. It is better if there is any possibility for reuse of it. What do you think? 

--> On bigger apps, we decided to separate by pages.

Reply:
For this version, I don't believe it is necessary. Is just a small test with your setups. If the tests get bigger and need to reuse those options, we can extract it.